### DISCUSSION TITLE
changing the directory colour in a ls command

### DISCUSSION THREAD

**Discussion by davidsi** [ohmyzsh/ohmyzsh#10493] [Help] [answered]
I find the colour cyan very hard to see so when my directories show up in cyan it's hard to read them. I can't figure out how to change that, and only that.  I tried modifying the theme, which I can do, the .zshrc file, which I can do, but no indicator as to how to modify the output of the 'ls' command

thx

**Comment by mcornella** [comment, marked as answer]
For macOS's and BSD in general you need to change the `LSCOLORS` variable. For GNU's `ls` (in Linux that's the one used) you need to change the `LS_COLORS` variable. They both use different syntax; you can look up [this generator](https://geoff.greer.fm/lscolors/) for help generating that. Important: both variables need to be `export` for `ls` to be able to find the variable.

Example for macOS:
```zsh
export LSCOLORS="exfxcxdxbxegedabagacad"
```

[This gist](https://gist.github.com/thomd/7667642) has more in-depth information.

You can also use [lsd](https://github.com/Peltoche/lsd) or [exa](https://github.com/ogham/exa) as a replacement for `ls` which have more coloring options.

**Comment by davidsi** [comment]
Thanks, I’ll try it
